patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js

```diff
@@ -79,15 +93,14 @@ if (siteAlertWrapper !== null) {
     if (emergencyAlerts !== null) {
       observer.disconnect();
     }
+
     hamburgerMenuAlertScrolling();
+
   });
   jsonApiObserver.observe(siteAlertWrapper, {
     childList: true
   });
 }
-else {
```

Comment:
When no alerts on a page, css takes care of the flyout height.  So, the else part is not necessary.